packages/@adobe/spectrum-css-temp/components/steplist/index.css

```diff
@@ -13,255 +13,277 @@ governing permissions and limitations under the License.
+  font-size: var(--spectrum-global-dimension-font-size-200);
 
-  box-sizing: content-box;
-  width: calc(calc(var(--spectrum-steplist-step-width) * 1.5) - calc(var(--spectrum-steplist-marker-diameter) * 2));
-  bottom: calc(calc(var(--spectrum-steplist-segment-height) / 2) * -1);
+  .spectrum-Steplist-item {
+    grid-template-columns: var(--spectrum-steplist-marker-box-width-xlarge) auto;
+    grid-column-gap: var(--spectrum-steplist-chevron-link-gap-xlarge);
+  }
+  .spectrum-Steplist-link {
+    grid-column-gap: var(--spectrum-steplist-number-label-gap-xlarge);
 
-  /* Default is dashed */
-  border-bottom-width: var(--spectrum-steplist-segment-height);
-  border-bottom-style: dashed;
+    .spectrum-Steplist-marker {
+      line-height: var(--spectrum-steplist-marker-diameter-xlarge);
+      width: var(--spectrum-steplist-marker-diameter-xlarge);
+      height: var(--spectrum-steplist-marker-diameter-xlarge);
+      border-radius: var(--spectrum-steplist-marker-diameter-xlarge);
+      font-size: var(--spectrum-global-dimension-font-size-150);
+    }
+  }
+  .spectrum-Steplist-chevron {
+    transform: scale(1.5);
+  }
 }
 
+
 /* First step overrides */
 .spectrum-Steplist-item:first-child {
-  .spectrum-Steplist-markerContainer {
-    left: 0;
-  }
-
-  .spectrum-Steplist-label {
-    left: 0;
-  }
-
+  grid-template-areas: "link";
+  grid-template-columns: auto;
   .spectrum-Steplist-segment {
     /* Not visible for first step */
     display: none;
   }
 }
 
-/* Last step overrides */
-.spectrum-Steplist-item:last-child {
-  .spectrum-Steplist-label {
-    left: auto;
-    right: 0;
 
-    transform: translateX(50%);
+/* vertical version */
+.spectrum-Steplist--vertical {
+  flex-direction: column;
+  gap: 0;
+  .spectrum-Steplist-item {
+    grid-template-areas: "chevron ."
+                         "link link";
+    justify-items: left;
+    &.is-navigable {
+      .spectrum-Steplist-segment {
+        border-style: solid;
+      }
+    }
   }
-
-  .spectrum-Steplist-markerContainer {
-    left: auto;
-    right: 0;
-
-    margin-left: 0;
-    margin-right: calc(calc(var(--spectrum-steplist-marker-hitArea) / 2) * -1);
+  .spectrum-Steplist-chevron {
+    display: none;
   }
-
   .spectrum-Steplist-segment {
-    right: var(--spectrum-steplist-marker-diameter);
-    left: auto;
-  }
-}
-
-.spectrum-Steplist-item:first-child,
-.spectrum-Steplist-item:last-child {
-  width: calc(calc(var(--spectrum-steplist-step-width) / 2) - calc(var(--spectrum-steplist-marker-diameter) * 2.5));
-}
-
-/* Single step */
-.spectrum-Steplist-item:only-child {
-  .spectrum-Steplist-label {
-    /* Be centered */
-    left: 50%;
-
-    /* Override last-child bits */
-    transform: translate(-50%);
-  }
-
-  .spectrum-Steplist-markerContainer {
-    /* Be centered */
-    left: 50%;
-
-    /* Override last-child bits */
-    margin-right: 0;
-    margin-left: calc(calc(var(--spectrum-steplist-marker-hitArea) / 2) * -1);
+    grid-area: chevron;
```

Comment:
if there is no chevron in vertical, let's rename the grid area to be more meaningful